chainio/clients/builder.go

```diff
@@ -206,6 +206,7 @@ func (config *BuildAllConfig) BuildELClients(
 		elContractBindings.Slasher,
 		elContractBindings.DelegationManager,
 		elContractBindings.StrategyManager,
+		nil,
```

Comment:
add a TODO comment for why this is nil?

Reply:
7110a2a1af0d56c03fe7bbea16e30f52e33b904f